src/main/scala/scorex/core/network/NetworkControllerSharedMessages.scala

```diff
@@ -0,0 +1,8 @@
+package scorex.core.network
+
+import scorex.core.network.message.MessageSpec
+import scala.reflect.runtime.universe.TypeTag
+
+trait NetworkControllerSharedMessages {
```

Comment:
shared with whom?

Reply:
In this case it is shared between  `NodeViewSynchronizer`, `PeerSynchronizer` and `NetworkController`